test/src/nyc-bin.js

```diff
@@ -689,4 +689,39 @@ describe('the nyc cli', function () {
       done()
     })
   })
+
+  it('allows alternative high and low watermarks to be configured', function (done) {
+    var args = [
+      bin,
+      '--watermarks.lines=90',
+      '--watermarks.lines=100',
+      '--watermarks.statements=30',
+      '--watermarks.statements=40',
```

Comment:
@JaKXz I see your point of view on this, and this would probably have been my inclination; however, this implementation matches `istanbul`'s:

https://github.com/gotwarlost/istanbul/issues/116

so I think it might be less surprising, it also allows us to kick the configuration directly along to the istanbul internals.